FBSnapshotTestCase/FBSnapshotTestController.h

```diff
@@ -67,11 +67,16 @@ extern NSString *const FBDiffedImageKey;
 @property (readwrite, nonatomic, assign) BOOL usesDrawViewHierarchyInRect;
 
 /**
- The directory in which referfence images are stored.
+ The directory in which reference images are stored.
  */
 @property (readwrite, nonatomic, copy) NSString *referenceImagesDirectory;
 
 /**
+ The directory in which failed snapshot images are stored.
```

Comment:
Can you also document the existence of the environmental variable and how this differs?